### PR TITLE
FashionMNIST/EMNIST Datamodules

### PR DESCRIPTION
**Pull request by matsumotosan**
## What does this PR do?

Fixes part of #839 

- `pl_bolts.datamodules.emnist_datamodule.EMNISTDataModule`
- `pl_bolts.datamodules.fashion_mnist_datamodule.FashionMNISTDataModule`

## Summary

- Remove under review warning tags for datamodules
- Clean docstrings

## Before submitting

- [x] Was this **discussed/approved** via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/lightning-bolts/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure your PR does **only one thing**, instead of bundling different changes together?
- [x] Did you make sure to **update the documentation** with your changes?
- [x] Did you write any **new necessary tests**? \[not needed for typos/docs\]
- [x] Did you verify new and **existing tests** pass locally with your changes?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/lightning-bolts/blob/master/CHANGELOG.md)?

## PR review

- [x] Is this pull request **ready for review**? (if not, please submit in draft mode)

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: tests/datamodules/test_datamodules.py [hunk 1]
```diff
@@ -112,7 +112,7 @@ def test_sr_datamodule(datadir):
 
 @pytest.mark.parametrize("split", ["byclass", "bymerge", "balanced", "letters", "digits", "mnist"])
 @pytest.mark.parametrize("dm_cls", [BinaryEMNISTDataModule, EMNISTDataModule])
```
Comment (otaj) [inline tests/datamodules/test_datamodules.py:114]: Can these tests also test `FashionMnistDataModule`? This applies to all the tests

**Review by otaj — APPROVED** [review]
I left one very minor comment, otherwise it looks good!

File: tests/datamodules/test_datamodules.py (hunk 1 above)
Comment (matsumotosan) [reply tests/datamodules/test_datamodules.py:114]: I don't think so. I think that these tests are only applicable to EMNIST and it's derivatives.

Comment (otaj) [reply tests/datamodules/test_datamodules.py:114]: Yes, you're right, sorry about that.

**Pull request merged**